src/pickup.cpp

```diff
@@ -1082,3 +1097,15 @@ int Pickup::cost_to_move_item( const Character &who, const item &it )
     // Keep it sane - it's not a long activity
     return std::min( 400, ret );
 }
+
+std::vector<Pickup::pickup_rect> Pickup::pickup_rect::list;
+
+Pickup::pickup_rect *Pickup::pickup_rect::find_by_coordinate( point p )
+{
+    for( pickup_rect rect : pickup_rect::list ) {
```

Comment:
You probably mean this
```suggestion
    for( pickup_rect &rect : pickup_rect::list ) {
```